tests/integration-tests/tests/update/test_update.py

```diff
@@ -529,3 +529,96 @@ def _test_max_vcpus(region, stack_name, vcpus):
 def _test_min_vcpus(region, stack_name, vcpus):
     asg_min_size = get_batch_ce_min_size(stack_name, region)
     assert_that(asg_min_size).is_equal_to(vcpus)
+
+
+@pytest.mark.dimensions("us-west-1", "c5.xlarge", "centos7", "sge")
+@pytest.mark.usefixtures("os", "instance")
+def test_sit_update_compute_instance(region, scheduler, pcluster_config_reader, clusters_factory, s3_bucket_factory):
```

Comment:
can we have 2 different test functions or parametrize this one as done in lustre tests so that the two tests are ran in parallel whenever possible?

Reply:
Done